tests/setup_test.go

```diff
@@ -1104,3 +1105,185 @@ func TestDirectRunVariables(t *testing.T) {
 		})
 	}
 }
+
+func TestDirectRunLogs(t *testing.T) {
+	config := `
+      {
+        runs: [
+          {
+            name: 'run01',
+            tasks: [
+              {
+                name: 'task01',
+                runtime: {
+                  containers: [
+                    {
+                      image: 'alpine/git',
+                    },
+                  ],
+                },
+                steps: [
+                  { type: 'clone' },
+                  { type: 'run', command: 'echo STEPLOG' },
+                ],
+              },
+            ],
+          },
+        ],
+      }
+    `
+
+	tests := []struct {
+		name   string
+		setup  bool
+		step   int
+		delete bool
+		err    error
+	}{
+		{
+			name: "test get log step 1",
+			step: 1,
+		},
+		{
+			name:  "test get log setup",
+			setup: true,
+		},
+		{
+			name: "test get log with unexisting step",
+			step: 99,
+			err:  errors.Errorf("internal server error"),
```

Comment:
This error isn't the expected one. I create #181 to fix this and return an ErrNotExists,